### PR TITLE
added Github repos stars count, notification bell on change

### PR DESCRIPTION
**Pull request by softvar**

### CODE REVIEW

File: Web/Github/git-repo-stars.1h.sh [hunk 1]
```diff
@@ -0,0 +1,118 @@
+#!/usr/local/bin/node
+
+/* Github repos stars count and Emoji on change :)
```
Comment (keithamus) [inline Web/Github/git-repo-stars.1h.sh]: Hey @softvar - again, for this you'll want to add some metadata. At least title, version, description & image please (https://github.com/matryer/bitbar#metadata for reference)

Comment (softvar) [inline Web/Github/git-repo-stars.1h.sh]: Hey @keithamus 
It's done here too :)

File: Web/Github/git-repo-stars.1h.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
+#!/usr/local/bin/node
+
+# <bitbar.title>Github Repo Stars</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Varun Malhotra</bitbar.author>
+# <bitbar.author.github>softvar</bitbar.author.github>
+# <bitbar.image>http://varunmalhotra.xyz/media/images/bitbar-plugin-github-repos-stars.png</bitbar.image>
+# <bitbar.desc>Github repos stars and Emoji notification 🔔 on change in stars count</bitbar.desc>
+# <bitbar.dependencies>node</bitbar.dependencies>
+# <bitbar.abouturl>http://varunmalhotra.xyz/blog/2016/02/bitbar-plugins-for-github-and-producthunt.html</bitbar.abouturl>
```
Comment (keithamus) [inline Web/Github/git-repo-stars.1h.sh]: `#` isn't a valid comment character in node. Which also made me notice that the extension for this file should be `.js` not `.sh`

Comment (softvar) [inline Web/Github/git-repo-stars.1h.sh]: My mistake, fixed now.

**Pull request merged**